exercises/concept/kurokos-clock/.docs/instructions.md

```diff
@@ -0,0 +1,48 @@
+# Instructions
+
+Your Japanese friend Kuroko is a streamer with an international appeal.
+Kuroko's streaming schedule is pretty intricate, each stream seemingly starts at a different time every day.
+
+He asks for your help in building a small application that will display his schedule in his viewers' local time zones: Kuroko's Clock.
+He has an additional requirement: the app should be able to display the time in two different languages: Japanese and US English.
+
+## 1. It's a date
+
+Define a function `showLocalDate : Locale -> Int -> Month -> Int -> String` which will display a (valid) local date.
+
+In the `US` locale, the format will be `M/D/Y`, while in the `JP` locale it will be `Y年M月D日`.
```

Comment:
Maybe expand this a bit, like you do for the time example? I found that my eyes didn't see the Y, M and D very easily in the string